db/migrate/20231127185636_create_standup_meeting_comments.rb

```diff
@@ -0,0 +1,10 @@
+class CreateStandupMeetingComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :standup_meeting_comments do |t|
+      t.string :name
```

Comment:
I think it's a good idea to change this to something like `section_name`. It took me some poking around to figure out what `name` meant.

Also is there any reason the `name` could every be empty? My understanding is that it's very important that it exists and that it maps to the different update types (today, yesterday, blockers) -- else it can't be displayed properly.

If my understanding is correct, then at minimum I think this field needs to be `null: false`. But also we could potentially think about making it an enum field?

Reply:
Thank you for the feedback. Changes made!